scripts/util/deploy-protocol-clients.js

```diff
@@ -36,7 +36,7 @@ async function deployProtocolClients(protocolClientArgs, maxPriorityFeePerGas, i
   // Deploy Protocol Client proxy contracts
   const protocolClientProxies = await deployProtocolClientProxies(protocolClientBeacons, maxPriorityFeePerGas);
 
-  // Cast the proxies to their implementation interfaces
+  // Cast the proxies to their implementation interfaces ?? ToDo: what is this even needed?
```

Comment:
That's a great question. I have checked all the places where we use this value and the correct seller BV address is calculated and attached before any calls are made. This function call (and consequently this cast-protocol-clients-proxies file) is unnecessary. Nice catch. (nothing saying to remove it here, just point it out for future revision)